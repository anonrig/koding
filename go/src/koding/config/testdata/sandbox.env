KONFIG_KLOUD_ENVIRONMENT=sandbox
KONFIG_KLOUD_KEYGENBUCKET=kodingdev-publiclogs
KONFIG_KLOUD_REGION=aws
KONFIG_KLOUD_REGISTERURL=https://sandbox.koding.com/kloud/kite
KONFIG_KLOUD_KONTROLURL=https://sandbox.koding.com/kontrol/kite
KONFIG_KLOUD_TUNNELURL=http://dev-t.koding.com