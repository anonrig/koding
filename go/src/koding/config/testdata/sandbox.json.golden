{
  "buckets": {
    "publiclogs": [
      {
        "environment": [
          "dev",
          "development",
          "sandbox",
          "devmanaged"
        ],
        "name": "kodingdev-publiclogs",
        "region": "us-east-1"
      }
    ]
  },
  "endpoints": {
    "ip": [
      {
        "environment": [
          "production",
          "managed"
        ],
        "url": "https://p.koding.com/-/ip"
      },
      {
        "environment": [
          "default",
          "dev",
          "development",
          "sandbox",
          "devmanaged"
        ],
        "url": "https://dev-p2.koding.com/-/ip"
      }
    ],
    "ipcheck": [
      {
        "environment": [
          "production",
          "managed"
        ],
        "url": "https://p.koding.com/-/ipcheck"
      },
      {
        "environment": [
          "default",
          "dev",
          "development",
          "sandbox",
          "devmanaged"
        ],
        "url": "https://dev-p2.koding.com/-/ipcheck"
      }
    ],
    "kd-latest": [
      {
        "environment": [
          "production",
          "managed"
        ],
        "url": "https://koding-kd.s3.amazonaws.com/production/latest-version.txt"
      },
      {
        "environment": [
          "default",
          "dev",
          "development",
          "sandbox",
          "devmanaged"
        ],
        "url": "https://koding-kd.s3.amazonaws.com/development/latest-version.txt"
      }
    ],
    "klient-latest": [
      {
        "environment": [
          "default",
          "dev",
          "development",
          "sandbox"
        ],
        "url": "https://koding-klient.s3.amazonaws.com/development/latest-version.txt"
      },
      {
        "environment": [
          "production"
        ],
        "url": "https://koding-klient.s3.amazonaws.com/production/latest-version.txt"
      },
      {
        "environment": [
          "managed"
        ],
        "url": "https://koding-klient.s3.amazonaws.com/managed/latest-version.txt"
      },
      {
        "environment": [
          "devmanaged"
        ],
        "url": "https://koding-klient.s3.amazonaws.com/devmanaged/latest-version.txt"
      }
    ],
    "kloud": [
      {
        "environment": [
          "dev",
          "development",
          "sandbox",
          "devmanaged"
        ],
        "url": "http://94.254.191.51:8090/kloud/kite"
      }
    ],
    "kontrol": [
      {
        "environment": [
          "dev",
          "development",
          "sandbox",
          "devmanaged"
        ],
        "url": "https://sandbox.koding.com/kontrol/kite"
      }
    ],
    "tunnelserver": [
      {
        "environment": [
          "dev",
          "development",
          "sandbox",
          "devmanaged"
        ],
        "url": "http:/dev-t.koding.com/kite"
      }
    ]
  },
  "routes": {
    "dev.koding.com": "127.0.0.1"
  }
}